PerformanceTests/MallocBench/MallocBench/mbmalloc.cpp

```diff
@@ -42,10 +42,7 @@ void* mbmalloc(size_t size)
 
 void* mbmemalign(size_t alignment, size_t size)
 {
-    void* result;
-    if (posix_memalign(&result, alignment, size))
-        return nullptr;
-    return result;
+    return aligned_alloc(alignment, size);
```

Comment:
I wouldn't touch MallocBench at all because it's a benchmark. I bet this _probably_ won't affect performance, but I don't know for sure, and we don't want to be wrong. It's OK for MallocBench to follow older patterns than the rest of WebKit.